docs/EXAMPLES.md

```diff
@@ -14,8 +14,8 @@
 
 ## setTimeout
 
-|           1 optional argument           |
-| :-------------------------------------: |
+| 1 optional argument                     |
```

Comment:
@pedrohyvo, out-of-curiosity, why did you change that?

Same question for all other appearences of that.

Reply:
@wlsf82 I've formatted the code using a VS code plugin `Markdown Table Prettifier`.

Reply:
done